addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-connection-service.js

```diff
@@ -124,6 +129,20 @@ class EnvironmentScConnectionService extends Service {
     return adjustedConnections;
   }
 
+  verifyAccess(requestContext, createdBy) {
+    const restrictAdminWorkspaceConnection =
+      this.settings.getBoolean(settingKeys.restrictAdminWorkspaceConnection) || false;
+    const uid = _.get(requestContext, 'principalIdentifier.uid');
+    const isAdmin = _.get(requestContext, 'principal.isAdmin');
+    if (restrictAdminWorkspaceConnection !== true) {
+      return true;
```

Comment:
Should we also check to see if user is an admin prior to returning true?

Reply:
Yes. As you suggested added in the prior returning true.